modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/view/ServiceTemplateHistoryVo.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+
+package org.eclipse.xpanse.modules.models.servicetemplate.view;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * Defines view object for service template history.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ServiceTemplateHistoryVo {
```

Comment:
We need a better name for this type. Maybe 'ServiceTemplateChangeInfo' ?
Also we can move it out of the view package.